src/main/java/net/openhft/chronicle/threads/Pauser.java

```diff
@@ -46,17 +51,24 @@ static boolean getSleepy() {
         return procs < MIN_PROCESSORS;
     }
 
+    /**
+     * Returns a {@link Pauser} that either yields, pauses, or does not wait at all, based on system capabilities.
+     * It selects the most appropriate pauser based on CPU availability and specified minimal busyness.
+     *
+     * @param minBusy the minimal busyness period in microseconds before yielding or pausing
+     * @return the most appropriate {@link Pauser}
+     */
     static Pauser yielding(int minBusy) {
         SleepyWarning.warnSleepy();
         return SLEEPY ? sleepy()
                 : BALANCED ? balanced()
-                : new TimeoutPauser(minBusy);
+                : new YieldingPauser(minBusy);
```

Comment:
why change?

Reply:
When you ask for a `yielding` you were getting a `TimeoutPauser` (for historical reasons) which was almost a duplicate of `YieldingPauser` I have since made TimeoutPauser just extend to YieldingPauser for removal.